applications/tutorials/chainHybrid/CMakeLists.txt

```diff
@@ -2,7 +2,6 @@ cmake_minimum_required(VERSION 3.12)
 project(chainHybrid)
 
 find_package(SceneCreator REQUIRED)
-find_package(SofaGui REQUIRED)
```

Comment:
You just remove the find_package() but you did not remove `SofaGui` in the `add_library()` line, meaning you still want to use it (and it works because this project is compiled alongside SOFA so it finds SofaGui by itself)
By the way, this tutorial still uses `sofa::gui::initSofaGui();` from.... `SofaGui` 🙊